src/utils/message.ts

```diff
@@ -0,0 +1,19 @@
+import dayjs from 'dayjs';
```

Comment:
https://how-to.dev/dayjs-vs-date-fns

+ 

https://date-fns.org/v2.25.0/docs/isSameDay